gulpfile.js

```diff
@@ -1,97 +1,84 @@
 // Gulpfile to build sonoma CLI
 
-const clean = require('gulp-clean');
-const gulp = require('gulp');
-const rimraf = require('rimraf');
-const sourcemaps = require('gulp-sourcemaps');
-const ts = require('gulp-typescript');
-const autocompleteTree = require('./scripts/autocomplete-tree');
+const clean = require("gulp-clean");
+const gulp = require("gulp");
+const rimraf = require("rimraf");
+const sourcemaps = require("gulp-sourcemaps");
+const ts = require("gulp-typescript");
+const autocompleteTree = require("./scripts/autocomplete-tree");
 
-let tsProject = ts.createProject('tsconfig.json');
+const tsProject = ts.createProject("tsconfig.json");
 
 //
 // General compile
 //
 
-gulp.task('clean', function (done) {
-  rimraf('dist', done);
+gulp.task("clean", function (done) {
+  rimraf("dist", done);
 });
 
-gulp.task('build-ts', function () {
-  let tsResult = gulp.src(['src/**/*.ts', 'typings/**/*.d.ts'])
-    .pipe(tsProject());
+gulp.task("build-ts", function () {
+  const tsResult = gulp.src(["src/**/*.ts", "typings/**/*.d.ts"]).pipe(tsProject());
```

Comment:
we can probably remove typings from here

Reply:
Good find. I also removed it from the doc.